src/Main.java

```diff
@@ -1,5 +1,64 @@
 public class Main {
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        // Задача 1
+        var dog = 8.0;
+        System.out.println( dog );
+        var cat = 3.6;
+        System.out.println( cat );
+        var paper = 763789;
+        System.out.println( paper );
+        // Задача 2
+        var dog1 = dog + 4;
+        System.out.println( dog1 );
+        var cat1 = cat + 4;
+        System.out.println( cat1 );
+        var paper1 = paper + 4;
+        System.out.println( paper1 );
+        // Задача 3
+        var dog2 = dog - 3.5;
+        System.out.println( dog2 );
+        var cat2 = cat - 1.6;
+        System.out.println( cat2 );
+        var paper2 = paper - 7639;
+        System.out.println( paper2 );
+        // Задача 4
+        var friend = 19;
+        System.out.println( friend );
+        var friend1 = 19 + 2;
+        System.out.println( friend1 );
+        var friend2 = friend1 / 7;
+        System.out.println( friend2 );
+        // Задача 5
+        var frog = 3.5;
+        System.out.println( frog );
+        var frog1 = frog * 10;
+        System.out.println( frog1 );
+        var frog2 = frog1 / 3.5;
+        System.out.println( frog2 );
+        var frog3 = frog2 + 4;
+        System.out.println( frog3 );
+        // Задача 6
+        var boxerOne = 78.2;
+        var boxerTwo = 82.7;
+        var result1 = boxerOne + boxerTwo;
+        System.out.println( result1 );
+        var result2 = boxerTwo - boxerOne;
+        System.out.println( result2 );
+        // Задача 7
+        result2 = boxerTwo - boxerOne;
+        System.out.println( result2 );
+        var result3 = boxerTwo % boxerOne;
+        System.out.println( result3 );
+        // Задача 8
+        var workingHours1 = 640;
+        var workingHours2 = 8;
+        var personal1 = workingHours1 / workingHours2;
+        System.out.println( " Всего работников в компании - " + personal1 + " человек ." );
+        var personal2 = personal1 + 94;
```

Comment:
```suggestion
        personal1 = personal1 + 94;
```
зачем создавать новую переменную, если можно поменять существующую?